container/test-app/wc/clientAPI.html

```diff
@@ -37,6 +37,7 @@ <h3>
         viewURL="./helloWorldWCSelfRegistered.js"
         webcomponent='{"selfRegistered": "true"}'
     ></luigi-container> -->
+      <script></script>
```

Comment:
Why do we need this empty script tag?

Reply:
good catch. missed it. applied suggestions 👍